R/palaeorotate.R

```diff
@@ -318,22 +320,32 @@ palaeorotate <- function(occdf, lng = "lng", lat = "lat", age = "age",
         which.min(abs(occdf[i, age] - rot_age))
       })]
 
-    # Search for matching longitude and  latitude
-    occdf$rot_lng <- sapply(seq_len(nrow(occdf)), function(i) {
-      # Extract closest longitude
-      base_model[which.min(abs(base_model[, c("lng")]  - occdf[i, lng])), 1]
-    }, simplify = TRUE)
+    # Set-up
+    # Convert to sf object and add CRS
+    occdf_sf <- sf::st_as_sf(x = occdf,
```

Comment:
st_as_sf and point_to_cell are now imported, so you don't need to specify the package and the double colon (but make sure to run document to add the imports to the NAMESPACE file).

Reply:
Done!